app/src/main/java/com/vci/vectorcamapp/core/presentation/components/tile/ActionTile.kt

```diff
@@ -29,12 +37,12 @@ fun ActionTile(
                 vertical = MaterialTheme.dimensions.paddingSmall
             )
             .customShadow(
-                color = MaterialTheme.colors.cardGlow.copy(alpha = 0.2f),
-                offsetX = MaterialTheme.dimensions.shadowOffsetSmall,
-                offsetY = MaterialTheme.dimensions.shadowOffsetSmall,
-                cornerRadius = MaterialTheme.dimensions.cornerRadiusMedium,
-                spread = MaterialTheme.dimensions.shadowSpreadSmall,
-                blurRadius = MaterialTheme.dimensions.shadowBlurSmall,
+                color = shadowColor,
+                offsetX = shadowOffsetX,
+                offsetY = shadowOffsetY,
+                cornerRadius = shadowCornerRadius,
+                spread = shadowSpread,
+                blurRadius = shadowBlurRadius,
```

Comment:
Not super sure about this custom shadow parameters being passed into the action tile...Why not just have the same shadow effect for all action tiles and just pass in a "hue" attribute?